.github/workflows/run-CI.yml

```diff
@@ -6,20 +6,24 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python: [3.7, 3.8, 3.9, '3.10']
+        python: [3.7, 3.8, 3.9, '3.10', 3.11]
```

Comment:
Please surround the value `3.11` with quotations marks (`'3.11'`) as otherwise the yaml compiler will cast this value to 3.1 instead of 3.11

Reply:
Thanks for the heads up I was not aware of that 👀 
Commit [e688555](https://github.com/WillianFuks/tfcausalimpact/pull/80/commits/e688555b6a4949624e4d8d0740b81dfec1debbd4) applies your suggestion ;)